src/EntraModuleBuilder.ps1

```diff
@@ -492,9 +492,9 @@ $($requiredModulesEntries -join ",`n")
             CompanyName = $($content.owners)
             FileList = @("$manifestFileName", "$moduleFileName", "$helpFileName")
             RootModule = "$moduleFileName"
-            Description = 'Microsoft Graph Entra PowerShell.'
-            DotNetFrameworkVersion = $([System.Version]::Parse('4.7.2'))
-            PowerShellVersion = $([System.Version]::Parse('5.1'))
+            Description = $content.EntraDescription
```

Comment:
```suggestion
            Description = $content.ModuleName
```

Reply:
Please update line 340 too. You might have missed it. https://github.com/microsoftgraph/entra-powershell/blob/6f5455b6ad1414f1c555b8cd1dcf348200e70ba1/src/EntraModuleBuilder.ps1#L340